src/Core/Dfe.Complete.Application/Projects/Queries/ListAllLocalAuthorities/ListAllProjectLocalAuthorities.cs

```diff
@@ -0,0 +1,49 @@
+using Dfe.Complete.Application.Common.Models;
+using Dfe.Complete.Application.Projects.Interfaces;
+using Dfe.Complete.Application.Projects.Models;
+using Dfe.Complete.Domain.Entities;
+using Dfe.Complete.Domain.Enums;
+using Dfe.Complete.Domain.Interfaces.Repositories;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Dfe.Complete.Application.Projects.Queries.ProjectsByLocalAuthority;
+
+public record ListAllProjectLocalAuthoritiesQuery(ProjectState? State = ProjectState.Active, ProjectType? Type = null)
```

Comment:
Really minor one but I think this should be List all projects by local authority
```suggestion
public record ListAllProjectsByLocalAuthoritiesQuery(ProjectState? State = ProjectState.Active, ProjectType? Type = null)
```